### ISSUE TITLE
Package request: PyBDSF

### ISSUE THREAD

**Issue by sabourke** [kernsuite/packaging#79]
https://github.com/lofar-astron/PyBDSF
This is used pretty widely within the lofar community and depended on by prefactor

**Comment by gijzelaerr** [comment]
yeah, i'm not sure why I forgot this one. Will add!

**Comment by sabourke** [comment]
Great!

**Comment by bennahugo** [comment]
This was previously in the metapackage "lofar". Seems they moved it
```
root@f224b2a5f781:/# apt-get install lofar
Reading package lists... Done
Building dependency tree       
Reading state information... Done
lofar is already the newest version (3.0.3-1kern1).
0 upgraded, 0 newly installed, 0 to remove and 14 not upgraded.
root@f224b2a5f781:/# pybdsf 
bash: pybdsf: command not found
```

**Comment by gijzelaerr** [comment]
pybdsf doesn't build with pybuild:
https://gist.github.com/gijzelaerr/644c4d6cde70e5ef026b6e84fa151c03

**Comment by gijzelaerr** [comment]
issue reported upstream https://github.com/lofar-astron/PyBDSF/issues/45

**Comment by sabourke** [comment]
Fixed at [sabourke/pybdsf](https://github.com/sabourke/pybdsf/)
The upstream setup.py was doing building whenever it was run. I moved the manual building and only call it during a build. This could probably get merged upstream.
I also include a nasty pybuild hook in debian/rules. If a module's name ends with "module" pybuild strips if off when using recent versions of python, see dhpython.interpreter.check_extname  I couldn't find much info on that but it breaks bdsf.nat.natgridmodule hence the nasty hack.
